sql/expression_value.h

```diff
@@ -1526,6 +1530,74 @@ struct UnknownRowValueInfo: public RowValueInfo {
     }
 };
 
+/*****************************************************************************/
+/* OR Expression Value Info                                                  */
+/*****************************************************************************/
+
+/** Expression Value info when we dont know which of two value info we will get 
+    With a Case for Example.
+*/
+
+struct ORExpressionValueInfo: public ExpressionValueInfoT<ExpressionValue> {
+
+    ORExpressionValueInfo(std::shared_ptr<ExpressionValueInfo> left, std::shared_ptr<ExpressionValueInfo> right) {
```

Comment:
We should avoid inlining functions in a class that's included all over the place.